source/concurrent/SynchronizedQueue.ooc

```diff
@@ -62,13 +62,10 @@ BlockedQueue: class <T> extends SynchronizedQueue<T> {
 		this _waitLock wake()
 	}
 	cancel: func { this _waitLock with(|| this _canceled = true) . wakeAll() }
-	wait: func (isOk := null as Bool*) -> T {
+	wait: func -> T {
 		result: T = null
 		this _waitLock lockWhen(func -> Bool { !this empty || this _canceled })
-		if (this _canceled) {
-			if (isOk)
-				isOk@ = false
-		} else
+		if (!this _canceled)
```

Comment:
I wish. `error Using different types in a ternary expression is forbidden. Find another way :)`
and if I change it to `null as T` I get a segfault.